java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -880,19 +881,61 @@ public Description matchBinary(BinaryTree tree, VisitorState state) {
     return Description.NO_MATCH;
   }
 
+  private boolean matchTestMethod(MethodInvocationTree methodTree, VisitorState state) {
+    Symbol receiverSymbol = ASTHelpers.getSymbol(methodTree.getMethodSelect());
+    String methodName = receiverSymbol.getSimpleName().toString();
+    for (PiranhaTestMethodRecord methodRecord : config.getTestMethodRecordsForName(methodName)) {
+      Optional<Integer> argumentIdx = methodRecord.getArgumentIdx();
+      if (argumentIdx.isPresent()) {
+        if (methodTree.getArguments().size() > argumentIdx.get()) {
+          ExpressionTree argTree = methodTree.getArguments().get(argumentIdx.get());
+          API api = getXPAPI(argTree, state);
+          if (api != API.UNKNOWN) {
+            return true;
+          }
+        }
+      } else {
+        for (ExpressionTree argTree : methodTree.getArguments()) {
```

Comment:
does this mean the flagTest API can occur at any index?

Reply:
Yes, but only if the argument index is not defined.